src/mame/drivers/houseball.cpp

```diff
@@ -0,0 +1,96 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/***************************************************************************
+
+    Skeleton driver for "House Ball", a electromechanic machine from the
+    Spanish company Olakoa with the same game mechanics as Taito's Ice Cold
+    Beer (but with different hardware). Uses bubble gums as prices.
```

Comment:
It’s priced in gum, or it gives gum as prizes?

Reply:
It gives gum as prizes. Fixed, thx!